services/orchestrator-agent/tests/conftest.py

```
@@ -101,3 +112,7 @@ def list(self):
 class MockDockerClient:
     def __init__(self):
         self.containers = MockContainerCollection()
+        self.images = type("obj", (object,), {"get": lambda self, id: type("img", (), {"save": lambda: [b"chunk"]})()})()
```

Ruff (0.16.2)

[warning] Unused lambda argument: `self`

(ARG005)